tests/PHPStan/Rules/Functions/CallToFunctionParametersRuleTest.php

```diff
@@ -1126,12 +1126,7 @@ public function testBug7211(): void
 
 	public function testBug5474(): void
 	{
-		$this->analyse([__DIR__ . '/../Comparison/data/bug-5474.php'], [
-			[
-				'Parameter #1 $data of function Bug5474\testData expects array{test: int}, *NEVER* given.',
-				26,
-			],
-		]);
+		$this->analyse([__DIR__ . '/../Comparison/data/bug-5474.php'], []);
```

Comment:
This would be the consequence of the change. (See https://github.com/phpstan/phpstan/issues/5474, https://phpstan.org/r/4ba9826f-8be3-4782-97ea-127ba76a9950)

At least to me that seems acceptable, as `never` is after all the bottom type.
This rather seems like a now undesired artifact from before `notIdentical.alwaysFalse` or similar.